src/js/components/AppListItemComponent.jsx

```diff
@@ -319,6 +325,20 @@ var AppListItemComponent = React.createClass({
     return <i className="icon icon-small app" title="Application"></i>;
   },
 
+  getLogsLink: function () {
+    var model = this.props.model;
+    const logsLink = Mustache.render(Config.appLogsLinkPattern, {
```

Comment:
I'm not sure to understand why you define this link outside of the condition

Reply:
Ack.